extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1195,12 +1195,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
         });
   }
 
+  /** @override */
   getPreconnectUrls() {
-    const urls = ['https://partner.googleadservices.com'];
     if (this.preloadSafeframe_) {
-      urls.push(SAFEFRAME_ORIGIN);
+      this.preconnect.preload(this.getSafeframePath());
```

Comment:
This switching us from just preconnecting to `tpc.googlesyndication.com` to now also asking the browser to preload the safeframe resource; this should be in the PR description.